### PR TITLE
add zip_export option for #16

### PR DESCRIPTION
**Pull request by tylerecouture**
My best stab. Sorry I don't know how to run or test this.. I'm not a TS/JS developer and started using github actions yesterday!  Apologies in advance for the amateur effort...

### CODE REVIEW

File: src/godot.ts [hunk 1]
```diff
@@ -232,6 +238,10 @@ async function move(zipPath: string): Promise<void> {
   await io.mv(zipPath, path.join(relativeProjectExportsPath, path.basename(zipPath)));
 }
 
+async function move_directory(dir: string): Promise<void> {
```
Comment (firebelley) [inline src/godot.ts:241]: Could you also make this function name `camelCase` instead of `snake_case`?

**Comment by firebelley** [comment]
Thanks for addressing the feedback! I just tested and it looks good. One thing I should note is these are the subdirectories that are created in the `exports` directory:
{7D184B05-5441-4B37-B4C6-2BF1B62ED211}

It's not a user friendly name at all, but if you're OK with that then I'll go ahead and merge. The reason these are numerical are for just in case you have several exports with the same name they won't overwrite each other.

**Comment by tylerecouture** [comment]
Cool!  I appreciate your responses and thanks for considering these changes!  I'm good with this for now =)  I can play around with it for my usecase and maybe as I get more familiar with this process I can PR something with a nicer naming convention and other tweaks?

**Comment by firebelley** [comment]
Yep I am fine with it as is, just wanted to make sure you were OK with it. I'm planning on reworking a lot of the core code at some point, so that may open up an opportunity to support better directory names.

Anyway, I am going to merge this and create a new release now!

**Pull request merged**